### PR TITLE
Features/cf submission

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app/routes/common-routes.js [hunk 1]
```diff
@@ -68,12 +68,14 @@
                 whenAsync('/register/NDB/new',           {templateUrl: 'views/forms/edit/edit-database.html',                  label:'New',  param:'true', resolveController: true,documentType :'NDB' , resolve : { securized : securize(null,true, true) }, }).
                 whenAsync('/register/VLR/new',           {templateUrl: 'views/forms/edit/edit-resource.html',                  label:'New',  param:'true', resolveController: true,documentType :'VLR' , resolve : { securized : securize(null, null, true) }, }).
                 whenAsync('/register/ORG/new',           {templateUrl: 'views/forms/edit/edit-organization.html',              label:'New',  param:'true', resolveController: true,documentType :'ORG' , resolve : { securized : securize(null, null, true) }, }).
-                
+                whenAsync('/register/SUB/new',           {templateUrl: 'views/forms/edit/edit-submission.html',                label:'New',  param:'true', resolveController: true,documentType :'SUB' , resolve : { securized : securize(['Administrator']) }, }).
```
Comment (sbilodeau) [inline app/routes/common-routes.js]: why `Administrator` only?

File: app/routes/common-routes.js [hunk 2]
```diff
@@ -68,12 +68,14 @@
...
                 whenAsync('/register/NDB/:identifier/edit',           {templateUrl: 'views/forms/edit/edit-database.html',                  label:'Edit',  param:'true', resolveController: true, documentType :'NDB' , resolve : { securized : securize(null,true, true) }, }).
                 whenAsync('/register/ORG/:identifier/edit',           {templateUrl: 'views/forms/edit/edit-organization.html',              label:'Edit',  param:'true', resolveController: true, documentType :'ORG' , resolve : { securized : securize(null, null, true) }, }).
                 whenAsync('/register/VLR/:identifier/edit',           {templateUrl: 'views/forms/edit/edit-resource.html',                  label:'Edit',  param:'true', resolveController: true, documentType :'VLR' , resolve : { securized : securize(null, null, true) }, }).
+                whenAsync('/register/SUB/:identifier/edit',           {templateUrl: 'views/forms/edit/edit-submission.html',                label:'Edit',  param:'true', resolveController: true, documentType :'SUB' , resolve : { securized : securize(['Administrator']) }, }).
```
Comment (sbilodeau) [inline app/routes/common-routes.js]: why `Administrator` only?

File: app/views/forms/edit/edit-submission.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                                                <div km-date name="date" ng-model="document.date"></div>
+                                            </div>
+                                </section>
+                                <section>
+                                    <div class="row">
+                                        <div class="col-md-5">
+                                            <div class="km-control-group" required name="notifications"
+                                                caption="Notifications">
+                                                <div document-selector ng-model="document.notifications"
+                                                    question="notifications" schema="(notification)" filter="true">
```
Comment (sbilodeau) [inline app/views/forms/edit/edit-submission.html]: what are the ()
```
 question="notifications" schema="(notification)" 
```

File: app/views/forms/edit/edit-submission.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
+﻿define(['app', 'lodash', 'views/forms/edit/edit', '../view/view-submission.directive',
+        'views/forms/edit/organization-selector'], function (app, _) {
+
+  app.controller("editSubmission", ["$scope", "$http", "$filter", "Thesaurus", "$q", "Enumerable", "$controller", "IStorage", "$location",
+                function ($scope, $http, $filter, Thesaurus, $q, Enumerable, $controller, storage, $location) {
+
+
+    $scope.path=$location.path();
```
Comment (sbilodeau) [inline app/views/forms/edit/edit-submission.js]: what this do?

File: app/routes/common-routes.js (hunk 2 above)
Comment (blaisefonseca) [reply app/routes/common-routes.js]: Submissions will only be entered by BCH staff rt?

File: app/views/forms/edit/edit-submission.html (hunk 3 above)
Comment (blaisefonseca) [reply app/views/forms/edit/edit-submission.html]: it was a hack to pass multiple schema's

File: app/views/forms/edit/edit-submission.js (hunk 4 above)
Comment (blaisefonseca) [reply app/views/forms/edit/edit-submission.js]: mistake, fixed

File: app/routes/common-routes.js (hunk 1 above)
Comment (sbilodeau) [reply app/routes/common-routes.js]: no they can be submitted by `any user` of the BCH.

Comment (blaisefonseca) [reply app/routes/common-routes.js]: done!

**Pull request merged**